exploration/text-vs-code.md

```diff
@@ -136,16 +209,93 @@ The current syntax includes some plain-ascii keywords:
 `input`, `local`, `match`, and `when`.
 
 The current syntax and active proposals include some sigil + name combinations,
-such as `:number`, `$var`, `|literal|`, `+bold`, and `@attr`.
+such as `:number`, `$var`, `|literal|`, `+bold`, `-bold`, and posibly `@attr`.
 
 The current syntax supports unquoted literal values as operands.
 
-Messages themselves are "simple strings" and must be considered to be a single
-line of text. In many containing formats, newlines will be represented as the local
-equivalent of `\n`.
+Messages themselves are "simple strings" and must be considered to be WYSIWYG.
```

Comment:
This is incorrect and the change greatly reduces the impact of the document IMO. The thing that is WYSIWYG is the pattern. In the case of simple messages, this is the whole message. But in the case of complex messages, what you see (`{#input $foo :number minimumFractionDigits=11}`) is not exactly what you get 😉 

The most important part of the original statement here is removed, which reminds readers that this message:

```
myMessage = {{
   {#input $var :number}
   {{You have {$var} message(s)}}
}}
```

Is actually this message in many storage formats:

```
myMessage = {{\n   {#input $var :number}\n   {{You have {$var} message(s)}}\n}}
```